misc/scripts/create-change-note.py

```diff
@@ -51,5 +52,6 @@
 with open(change_note_file, "w") as f:
     f.write(change_note)
 
-# Open the change note file in VSCode, reusing the existing window if possible
-os.system(f"code -r {change_note_file}")
+editor = os.environ.get('EDITOR', 'code')
```

Comment:
```suggestion
editor = os.environ.get('EDITOR', 'code -r')
```
Might make sense to include the `-r` flag here as well (though the rest of the changes make it unlikely to be needed).